web/src/layout/StackedLayout.vue

```diff
@@ -28,8 +30,19 @@ import Loader from "../components/Loader.vue"
 import { provideClient } from "@urql/vue"
 import client from "@/graph/client"
 import { init } from "@/services/language"
+import { ref } from "vue"
+import SettingContent from '@/pages/setting/SettingContent.vue'
+
 provideClient(client)
 
+const profileSettingIsShow = ref(false);
+
+function profileToggle() {
+    console.log('profileToggle')
+    profileSettingIsShow.value = !profileSettingIsShow.value;
```

Comment:
Not a fan of this syntax and variable naming. 

Use local variables as much as possible and handle toggles in template `@click="showSettings = !showSettings"`

Reply:
ok can I still make it like this
const settingsToggle = () => profileSettingIsShow.value = !profileSettingIsShow.value;

Reply:
No thanks. 

Keep it in template where it needs to be. Shorten the variable names as well.